app/views/pages/cookies.html.erb

```diff
@@ -1,156 +1,159 @@
-                  </td>
-                  <td>_ga 2 years<br/>
-                    _gid 24 hours</td>
-                </tr>
-
-                <tr>
-                  <td>_gat</td>
-                  <td>Used to manage the rate at which page view requests are made</td>
-                  <td>10 minutes</td>
-                </tr>
-              </tbody>
-            </table>
-          </div>
-        <% end %>
+      <%= render "govuk_publishing_components/components/govspeak", {
+      } do %>
+        <p>Cookies are files saved on your phone, tablet or computer when you visit a website.</p>
+        <p>We use cookies to store information about how you use the data.gov.uk website, such as the pages you visit.</p>
 
-        <%= render "govuk_publishing_components/components/radio", {
-          name: "cookies-usage",
-          inline: false,
-          heading: "Cookies that measure website use",
-          hint: cookies_usage_hint,
-          items: [
-            {
-              value: "on",
-              text: "Use cookies that measure my website use"
-            },
-            {
-              value: "off",
-              text: "Do not use cookies that measure my website use"
-            }
-          ]
-        } %>
-
-        <% cookies_campaigns_hint = capture do %>
-          These cookies may be set by third party websites. We don't currently use any cookies in this category.
-        <% end %>
+      <% end %>
 
-        <%= render "govuk_publishing_components/components/radio", {
-          name: "cookies-campaigns",
-          inline: false,
-          heading: "Cookies that help with our communications and marketing",
-          hint: cookies_campaigns_hint,
-          items: [
-            {
-              value: "on",
-              text: "Use cookies that help with communications and marketing"
-            },
-            {
-              value: "off",
-              text: "Do not use cookies that help with communications and marketing"
-            }
-          ]
-        } %>
-
-        <%= render "govuk_publishing_components/components/radio", {
-          name: "cookies-settings",
-          inline: false,
-          heading: "Cookies that remember your settings",
-          hint: "These cookies do things like remember your preferences and the choices you make, to personalise your experience of using the site.",
-          items: [
-            {
-              value: "on",
-              text: "Use cookies that remember my settings on the site"
-            },
-            {
-              value: "off",
-              text: "Do not use cookies that remember my settings on the site"
-            }
-          ]
-        } %>
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Cookie settings",
+        padding: true,
+      } %>
 
+      <div class="cookie-settings__no-js">
```

Comment:
Non blocking comment - is this `<div class="cookie-settings__no-js">` element doing the same thing as a `noscript` element?

Reply:
Turns out it does! Amending and pushing now.